docs/data/material/components/timeline/LeftAlignedTimeline.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Timeline from '@mui/lab/Timeline';
+import TimelineItem from '@mui/lab/TimelineItem';
+import TimelineSeparator from '@mui/lab/TimelineSeparator';
+import TimelineConnector from '@mui/lab/TimelineConnector';
+import TimelineContent from '@mui/lab/TimelineContent';
+import TimelineDot from '@mui/lab/TimelineDot';
+import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
+
+export default function OppositeContentTimeline() {
+  return (
+    <React.Fragment>
+      <Timeline
+        sx={{
+          '& .MuiTimelineOppositeContent-root': {
```

Comment:
```suggestion
          [`& .${timelineOppositeContent.root}`]: {
```
Let's use the class object. x the others.